docs/writing-docs/doc-blocks.md

```diff
@@ -17,9 +21,39 @@ Storybook Docs automatically generates component props tables for components in
   />
 </video>
 
+This is extremely useful, but it can be further expanded. Additional information can be added to the component to better document it:
+
+```js
+// Button.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({isDisabled,content}) {
+  return (
+    <button disabled={isDisabled}>{content}</button>
+  );
+}
+
+Button.propTypes = {
+  /**
+   Checks if the button should be disable
+  */
+  isDisabled: PropTypes.bool.isRequired,
+  /**
+  The display content of the button
+  */
+  content: PropTypes.string.isRequired,
+};
+
+```
+
+By including the additional information, the args tables will be updated. Offering a richer experience for any stakeholders involved.
```

Comment:
I guess it is "args table" (not plural) right?

Reply:
You're correct @tmeasday , fixed with the latest commit.